all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -40,29 +40,26 @@ metadata:
     name: jaeger-infra
     jaeger-infra: jaeger-template-all-in-one
 objects:
-- apiVersion: v1
-  kind: DeploymentConfig
+- apiVersion: extensions/v1beta1
```

Comment:
Good question. I *think* this was new in 1.5, but I'm not 100% sure. This is the feature in question: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/